tests/e2e/restart_test.go

```diff
@@ -84,7 +84,7 @@ func (s *RestartTestData) TestVolumeDetached(c *C) {
 	c.Assert(err, IsNil, Commentf(out))
 
 	// 4. Verify detached status. Volume should be detached (within the timeout)
-	status = verification.VerifyDetachedStatus(s.volumeName, s.config.DockerHosts[1], s.config.EsxHost)
+	status = verification.PollDetachedStatus(s.volumeName, s.config.DockerHosts[1], s.config.EsxHost)
```

Comment:
Actually, the admin CLi and docker cli ways of getting the volume status is the same - from the KV. So there's no difference in getting the status from the ESX side. Just doing a docker inspect is sufficient.  Which is why I was pointing to the newly added function. An instantaneous result is something that can change and doesn't require needing multiple checks (which are based off the same source anyway).